packages/calypso-build/typescript/ts-package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig",
+
+	"compilerOptions": {
+		// Allow composed projects
+		"composite": true,
+		"disableReferencedProjectLoad": true,
+
+		// Generate declarations and sourcemaps
+		"declaration": true,
+		"declarationMap": true,
+		"sourceMap": true,
+
+		// Do not emit helpers, rely on importing tslib
+		"importHelpers": true,
+		"noEmitHelpers": true,
+		"skipLibCheck": true,
+
+		// Target modern JS
+		"lib": [ "ES2020", "DOM" ],
+		"module": "ES2020",
+		"target": "ES2020",
+		"esModuleInterop": true,
+		"jsx": "react",
+
+		// Strict checks
+		"strict": true,
+		"isolatedModules": true,
```

Comment:
`isolatedModules` isn't a "strict" check.

Did you consider the any of the lint configuration? I don't see any enabled here.

Reply:
> isolatedModules isn't a "strict" check

Yeah, but I couldn't find a better "group". Maybe I should rename it to `// Code style`. Or drops the comments.

> Did you consider the any of the lint configuration?

I did remove it because `eslint` is already checking all the lint options we had enabled.